deployment/copilot/devDeploy.sh

```diff
@@ -76,39 +143,42 @@ printf "The following exports were added from CDK:\n%s\n" "$found_exports"
 
 cd ../../copilot
 
+# Reset AWS_DEFAULT_REGION as this will cause conflict with Copilot when interacting with multiple environments within the same AWS account
```

Comment:
If you have a bit more detail on why, it would be great to add it.  As it is, I'm confused.